eboutic/views.py

```diff
@@ -175,6 +176,20 @@ def get_context_data(self, **kwargs):
         kwargs["et_request"]["PBX_TIME"] = str(
             datetime.now().replace(microsecond=0).isoformat("T")
         )
+        kwargs["et_request"]["PBX_BILLING"] = dict2xml(
+            self.basket.user.customer.get_billing_info()
+        )
+        kwargs["et_request"]["PBX_SHOPPINGCART"] = dict2xml(
+            {
+                "shoppingcart": {
+                    "total": {
+                        "totalQuantity": 99
```

Comment:
You should probably precise in a comment that this value comes directly from the documentation and the protocol. A good idea when such hard-coded value must be used is to put it in a variable, such as `3DSV2_MAX_TOTAL_QUANTITY`, to show that you didn't choose this value arbitrarily, and hint future developer **not** to change it as they please.